tests/platform_helper/providers/test_copilot.py

```diff
@@ -7,53 +6,28 @@
 from botocore.exceptions import ClientError
 from moto import mock_aws
 
-from dbt_platform_helper.exceptions import AddonNotFoundError
-from dbt_platform_helper.exceptions import AddonTypeMissingFromConfigError
-from dbt_platform_helper.exceptions import InvalidAddonTypeError
-from dbt_platform_helper.exceptions import ParameterNotFoundError
+from dbt_platform_helper.exceptions import SecretNotFoundError
 from dbt_platform_helper.providers.copilot import CreateTaskTimeoutError
+from dbt_platform_helper.providers.copilot import _normalise_secret_name
 from dbt_platform_helper.providers.copilot import connect_to_addon_client_task
 from dbt_platform_helper.providers.copilot import create_addon_client_task
 from dbt_platform_helper.providers.copilot import create_postgres_admin_task
-from dbt_platform_helper.providers.secrets import SecretNotFoundError
-from dbt_platform_helper.providers.secrets import (
-    _normalise_secret_name as normalise_secret_name,
-)
-from dbt_platform_helper.providers.secrets import get_addon_type
-from dbt_platform_helper.providers.secrets import get_parameter_name
 from tests.platform_helper.conftest import NoSuchEntityException
-from tests.platform_helper.conftest import add_addon_config_parameter
 from tests.platform_helper.conftest import expected_connection_secret_name
-from tests.platform_helper.conftest import mock_parameter_name
 from tests.platform_helper.conftest import mock_task_name
 
 env = "development"
 
 
-@pytest.mark.parametrize(
-    "test_string",
-    [
-        ("app-rds-postgres", "APP_RDS_POSTGRES"),
-        ("APP-POSTGRES", "APP_POSTGRES"),
-        ("APP-OpenSearch", "APP_OPENSEARCH"),
-    ],
-)
-def test_normalise_secret_name(test_string):
-    """Test that given an addon name, normalise_secret_name produces the
-    expected result."""
-
-    assert normalise_secret_name(test_string[0]) == test_string[1]
-
-
 @mock_aws
 @patch(  # Nested function within provider function
-    "dbt_platform_helper.providers.copilot.get_postgres_connection_data_updated_with_master_secret",
+    "dbt_platform_helper.providers.secrets.SecretsManager.get_postgres_connection_data_updated_with_master_secret",
     return_value="connection string",
 )
 def test_create_postgres_admin_task(mock_update_parameter, mock_application):
 
     addon_name = "dummy-postgres"
-    master_secret_name = f"/copilot/{mock_application.name}/{env}/secrets/{normalise_secret_name(addon_name)}_RDS_MASTER_ARN"
+    master_secret_name = f"/copilot/{mock_application.name}/{env}/secrets/{_normalise_secret_name(addon_name)}_RDS_MASTER_ARN"
```

Comment:
It doesn't feel right to be using the production code/private method in the test like this...
```suggestion
    master_secret_name = f"/copilot/{mock_application.name}/{env}/secrets/DUMMY_POSTGRESS_RDS_MASTER_ARN"
```